.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -1,16 +1,18 @@
 ---
 name: Feature request
-about: Suggest an idea for Azeroth Pilot Reloaded
+about: Suggest an idea for Azeroth Pilot Reloaded.
 title: "[FEATURE] - "
-labels: "Feature"
+labels: Feature
 assignees: ''
 
 ---
 
-### Please describe the issues and/or needs to which your feature request relates
+### Please describe the issues and/or needs to which your feature request relates to.
```

Comment:
This change makes the sentence grammatically incorrect. It was correct in the previous iteration. One should not end a sentence with a preposition.

Reply:
alright